internal/cacheinvalidator/cache_invalidator.go

```diff
@@ -83,11 +86,11 @@ func NewCacheController(ds storage.OpenFGADatastore, cache storage.InMemoryCache
 	return c
 }
 
-func (c *InMemoryCacheController) DetermineInvalidation(
+func (c *InMemoryCacheInvalidator) GetLastWriteAndInvalidate(
 	ctx context.Context,
 	storeID string,
 ) time.Time {
-	_, span := tracer.Start(ctx, "cacheController.DetermineInvalidation", trace.WithAttributes(attribute.Bool("cached", false)))
+	_, span := tracer.Start(ctx, "cacheInvalidator.GetLastWriteAndInvalidate", trace.WithAttributes(attribute.Bool("cached", false)))
```

Comment:
I've never received proof from the community that someone is using the traces. It seems that only us, the maintainers, find them useful. Therefore, renaming.